it/uk/gov/hmrc/agentaccesscontrol/support/BaseISpec.scala

```diff
@@ -17,51 +17,50 @@
 package uk.gov.hmrc.agentaccesscontrol.support
 
 import com.github.tomakehurst.wiremock.client.WireMock._
-import org.scalatestplus.play.OneServerPerSuite
+import org.scalatestplus.play.{OneAppPerSuite, OneServerPerSuite}
 import play.api.test.FakeApplication
 import uk.gov.hmrc.agentaccesscontrol.StartAndStopWireMock
 import uk.gov.hmrc.agentaccesscontrol.model.{Arn, MtdSaClientId}
 import uk.gov.hmrc.domain.{AgentCode, SaAgentReference, SaUtr}
-import uk.gov.hmrc.play.http.HeaderCarrier
 import uk.gov.hmrc.play.test.UnitSpec
 
-abstract class BaseISpec extends UnitSpec
-    with StartAndStopWireMock
-    with StubUtils
-    with OneServerPerSuite {
-
-  private val configDesHost = "microservice.services.des.host"
-  private val configDesPort = "microservice.services.des.port"
-  private val configAuthHost = "microservice.services.auth.host"
-  private val configAuthPort = "microservice.services.auth.port"
-  private val ggProxyHost = "microservice.services.government-gateway-proxy.host"
-  private val ggProxyPort = "microservice.services.government-gateway-proxy.port"
-  private val auditingHost = "auditing.consumer.baseUri.host"
-  private val auditingPort = "auditing.consumer.baseUri.port"
-  private val agenciesHost = "microservice.services.agencies-fake.host"
-  private val agenciesPort = "microservice.services.agencies-fake.port"
-  private val relationshipsHost = "microservice.services.agent-client-relationships.host"
-  private val relationshipsPort = "microservice.services.agent-client-relationships.port"
-
-  implicit val hc = HeaderCarrier()
+abstract class WireMockISpec extends UnitSpec
+  with StartAndStopWireMock
+  with StubUtils {
+
+  protected val wireMockAppConfiguration: Map[String, Any] = Map(
+    "microservice.services.des.host" -> wiremockHost,
```

Comment:
I"m sure we've gone back and forth on whether these are constants or not a couple times. This is definitely preferable IMO.